apps/web/vibes/soul/primitives/products-carousel/index.tsx

```diff
@@ -19,11 +19,12 @@ export type CarouselProduct = CardProduct;
 interface Props {
   products: Streamable<CarouselProduct[]>;
   className?: string;
-  emptyStateTitle?: string;
-  emptyStateSubtitle?: string;
+  emptyStateTitle?: Streamable<string | null>;
```

Comment:
@migueloller Here's where I need clarification. In our discussion on Slack, you mentioned that primitives would get normal values in their props, but if this prop gets past the value from its parent, wouldn't it also need to be Streamable?

Reply:
This primitive is a bit more section-y than others. It handles its own skeleton, etc. so it makes sense for it to take streamable values. It's more about whether it makes sense or not for the component to have a fallback. For example, a skeleton shouldn't take a streamable value because what would the fallback for a skeleton be?